src/Grpc.AspNetCore.Server/Internal/PipeExtensions.cs

```diff
@@ -32,21 +32,27 @@ internal static class PipeExtensions
         private const int MessageDelimiterSize = 4; // how many bytes it takes to encode "Message-Length"
         private const int HeaderSize = MessageDelimiterSize + 1; // message length + compression flag
 
-        public static Task WriteMessageAsync<TResponse>(this PipeWriter pipeWriter, TResponse response, GrpcServiceOptions serviceOptions, Func<TResponse, byte[]> serializer, WriteOptions writeOptions)
+        private static readonly Status MessageCancelledStatus = new Status(StatusCode.Internal, "Incoming message cancelled.");
+        private static readonly Status AdditionalDataStatus = new Status(StatusCode.Internal, "Additional data after the message received.");
+        private static readonly Status IncompleteMessageStatus = new Status(StatusCode.Internal, "Incomplete message.");
+        private static readonly Status SendingMessageExceedsLimitStatus = new Status(StatusCode.ResourceExhausted, "Sending message exceeds the maximum configured message size.");
+        private static readonly Status ReceivedMessageExceedsLimitStatus = new Status(StatusCode.ResourceExhausted, "Received message exceeds the maximum configured message size.");
```

Comment:
FTR https://github.com/grpc/grpc/blob/master/doc/statuscodes.md

Reply:
Yeah I'm aware of that page and used it as a guide. This is the correct status, right?